fe/fe-core/src/main/java/com/starrocks/sql/Explain.java

```diff
@@ -819,12 +829,13 @@ static void buildCostEstimate(StringBuilder sb, OptExpression optExpression, int
 
         if (optExpression.getStatistics().getColumnStatistics().values().stream()
                 .allMatch(ColumnStatistic::isUnknown)) {
-            buildOperatorProperty(sb, "Estimates: {" +
-                    "row: " + (long) optExpression.getStatistics().getOutputRowCount() +
+            buildOperatorProperty(sb, "Estimates: {cost: " + optExpression.getCost() +
```

Comment:
cost, relative to other attributes, is a secondary attribute and should be placed at the end